spec/javascript/museum_test.js

```diff
@@ -1,23 +1,21 @@
+import Museum from '../../app/javascript/components/Museum';
+
 describe('Museum', () => {
   let name,
       wrapper;
 
   beforeEach(() => {
     jasmineEnzyme();
-    onClick = jasmine.createSpy('onClick spy'); //<-- this is new
+    // onClick = jasmine.createSpy('onClick spy');
     wrapper = mount(
       <Museum
-
         name = "Boston Children's Museum"
-
       />
     );
   });
 
-...
 //  new test
   it('should render the Museum component', () => {
-    expect(wrapper.find()).toBePresent();
-    name = "Boston Children's Museum"
+    expect(wrapper.find(Museum)).toBePresent();
   });
```

Comment:
Still not enough, you are just testing that the component exists. You have to test that "Boston Children's Museum" is present.